scripts/test_release.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+OS_LIST=(
+    centos:7
+    centos:8
+    fedora:28
+    ubuntu:16.04
+    ubuntu:18.04
+)
+
+FAILED=()
+RET_VALUE=0
+
+# We'll use this simple tokenized Dockerfile.
+# https://serverfault.com/a/72511
+IFS='' read -r -d '' TOKENIZED <<"EOF"
+FROM {{OS}}
+
+ENV DEBIAN_FRONTEND noninteractive
+{{PACMAN}}
+ADD https://algorand-releases.s3.us-east-1.amazonaws.com/channel/stable/install_stable_linux-amd64_2.0.1.tar.gz /tmp
+
+RUN \
+  set -eux; \
+  mkdir /opt/installer ; \
+  cd /opt/installer ; \
+  tar xvf /tmp/install*tar.gz ; \
+  ./update.sh -i -c stable -p /opt/algorand/node -d /opt/algorand/node/data -n ;
+
+WORKDIR /opt/algorand/node
+RUN ["./goal", "node", "start", "-d", "data"]
```

Comment:
I think if you replaced this `RUN` with `CMD ["/opt/algorand/node/algod", "-v"]` you wouldn't need to specify the bash command when starting the node. Also you shouldn't need to start the node for this test.

Reply:
Good to know, will change.